smpe/bld/SMPMCS.txt

```diff
@@ -11,7 +11,7 @@
  * Copyright Contributors to the Zowe Project. 2019, [YEAR]
  */
   .
-++VER(Z038,C150,P115) /* zOS, CICS, IMS & DB2 */
+++VER(Z038) /* zOS, CICS, IMS & DB2 */
```

Comment:
Should the comment be updated to remove CICS etc?

Reply:
good catch, fixed